lib/dal/src/management/prototype.rs

```diff
@@ -251,9 +254,12 @@ impl ManagerComponent {
         kind: &str,
         views: &HashMap<ViewId, String>,
     ) -> ManagementPrototypeResult<ManagerComponent> {
+        let component = ManagedComponent::new(ctx, component_id, kind, views).await?;
```

Comment:
AFAICT, ManagerComponent no longer needs to exist--incoming_connections is literally the only difference. If Component turns out to be a bad name, we could rename `ManagedComponent` -> `ManagementComponent` and remove `ManagerComponent` entirely.